### ISSUE TITLE
Empty map causes odd mapref error

### ISSUE THREAD

**Issue by robander** [dita-ot/dita-ot#1549]
I have an empty map that is pulled into my primary map. The map is generated, and in this case, happened to have no content, just
```xml
<map title="sample title">
</map>
```

I've added a reference to this in hierarchy.ditamap:
```xml
<topicref href="copy.ditamap" format="ditamap"/>
```

With the map as shown above, I get this error:
>  [mapref] [DOTX031E][ERROR]: The file copy.ditamap is not available to resolve link information.  The location of this problem was at (File = C:\DITA-OT1.8\samples\hierarchy.ditamap, Element = topicref:20;31:49)

If I add anything into copy.ditamap -- even just a topicref with a navtitle, and no topic -- the error goes away.

**Comment by robander** [comment]
Still an issue in 2.2.4.

Appears to be caused by this line: https://github.com/dita-ot/dita-ot/blob/develop/src/main/xsl/preprocess/maprefImpl.xsl#L88
```
 <xsl:when test="empty($file) or empty($file/*/*)">
```
If the document exists on the file system, it's place in `$file`. So if `empty($file)`, it makes sense to say the file is not available. I'm not sure though why we check for `/*/*` - this means that a file which exists, but only has a root element, will generate the "file is not available" message. The same file with a meaningless `<topicmeta/>` or `<data/>` child will not generate the message. I think we should probably change that check to `/*` just to check for a root node - though without that, `empty($file)` will probably be triggered?

**Comment by robander** [comment]
Fixed for 2.2.5.
